scss/colors.scss

```diff
@@ -0,0 +1 @@
+$earls-green: #BED23C;
```

Comment:
Add all the colors that are indicated on this trello card, aswell as the sizes: https://trello.com/c/qp5nMqvV/11-setup-del-proyecto